lib/idx/idxState/v1/idxResponseParser.ts

```diff
@@ -11,22 +11,22 @@
  */
 
 /* eslint-disable max-len */
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-nocheck
-import { OktaAuthIdxInterface } from '../../types';    // auth-js/types
+
+import { MinimalOktaAuthIdxInterface, IdxResponse, IdxRemediation, IdxContext } from '../../types';    // auth-js/types
+import { IdxActions } from '../../types/idx-js';
 import { generateRemediationFunctions } from './remediationParser';
 import generateIdxAction from './generateIdxAction';
 import { jsonpath } from '../../../util/jsonpath';
 import { AuthSdkError } from '../../../errors';
 
-const SKIP_FIELDS = Object.fromEntries([
```

Comment:
Using `Object.fromEntries` causes TS error: 
`Property 'fromEntries' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2019' or later.`
Changing `lib` in `tsconfig` doesn't help, so refactored code to not use `fromEntries`